doc/model/train-se-atten.md

```diff
@@ -161,6 +165,16 @@ We highly recommend using the version 2.0 of the attention-based descriptor `"se
       "set_davg_zero": false
 ```
 
+The `tebd_input_mode` can take `"concat"` and `"strip"` as values. The default value in `"se_atten_v2"` is `"strip"`. When `tebd_input_mode` is set to `"strip"`, the embedding matrix $\mathcal{G}^i$ is constructed as:
```

Comment:
I don't think `se_atten_v2` has a `tebd_input_mode` argument and supports `concat`. FYI @iProzd

Reply:
I notice that the implementation of `:se_attn_v2"` is not the same in TensorFlow backend and PyTorch backend, I have modified the description, please review. @njzjz @iProzd 
<img width="1028" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/100290172/78701816-c671-4b0b-a2d2-c238b7d3f880">